__tests__/data/dynamicConfig_input.json

```diff
@@ -0,0 +1,106 @@
+{
+  "request": {
+    "body": [
+      {
+        "destination": {
+          "Config": {
+            "appId": "{{ message.traits.appId || \"<APP_ID>\" }}"
+          }
+        },
+        "metadata": {
+          "jobId": 2
+        },
+        "message": {
+          "anonymousId": "sampath",
+          "channel": "web",
+          "context": {},
+          "traits": {
+            "anonymousId": "sampath",
+            "email": "sampath@gmail.com",
+            "appId": "testAppId"
+          }
+        }
+      },
+      {
+        "destination": {
+          "Config": {
+            "appId": [{"from" :"{{ message.traits.appId || \"<APP_ID>\" }}"} ,{ "to": "{{ message.traits.appId || \"<APP_ID>\" }}"}]
```

Comment:
give different values. Both are same objects.

Reply:
and from to should be within the same object. Create another object with from to with different values

Reply:
@abandonned make sure the test cases fully cover the module.

Reply:
Have added some more test cases . Please check .

Reply:
@ruchiramoitra  , as per suggestion of @saikumarrs  we have made code more generic to support all kinds of possibilities. Please check the `dynamicConfig.js `file as well.